static/font/font.css

```diff
@@ -51,29 +53,77 @@
     font-family: 'Roboto Mono';
     font-style: normal;
     font-weight: 500;
-    src: local('Roboto Mono Medium'), url('RobotoMono-Medium.woff') format('woff');
+    src: local('Roboto Mono Medium'), url('roboto/RobotoMono-Medium.woff') format('woff');
     }
     
 
     @font-face {
     font-family: 'Roboto Mono';
     font-style: italic;
     font-weight: 500;
-    src: local('Roboto Mono Medium Italic'), url('RobotoMono-MediumItalic.woff') format('woff');
+    src: local('Roboto Mono Medium Italic'), url('roboto/RobotoMono-MediumItalic.woff') format('woff');
     }
     
 
     @font-face {
     font-family: 'Roboto Mono Bold';
     font-style: normal;
     font-weight: 700;
-    src: local('Roboto Mono Bold'), url('RobotoMono-Bold.woff') format('woff');
+    src: local('Roboto Mono Bold'), url('roboto/RobotoMono-Bold.woff') format('woff');
     }
     
 
     @font-face {
     font-family: 'Roboto Mono';
     font-style: italic;
     font-weight: 700;
-    src: local('Roboto Mono Bold Italic'), url('RobotoMono-BoldItalic.woff') format('woff');
-    }
\ No newline at end of file
+    src: local('Roboto Mono Bold Italic'), url('roboto/RobotoMono-BoldItalic.woff') format('woff');
+    }
+
+
+/**** INTER ****/
+
+/* Inter */
+ @font-face {
+    font-family: "Inter";
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url("inter/Inter-Regular.woff2?v=3.19") format("woff2"),
```

Comment:
do we have to use non-standard fonts? I can understand for monospace. 

we can maybe support non-standard non-monospace fonts if we're sure that the fonts will load only after everything else loads first (not in parallel, but explicitly last).

Reply:
I have made some tests and I think we can avoid use non-standard `sans-serif` font if we use a good font fallback system such as: `ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"`
    
But besides the fonts, I can understand your wish to improve the website loading speed, I've pushed some optimizations that give us good results.

- preloading rendering javascript ressources
- preloading `Roboto` font and make a good use of HTTP/2 request multiplexing.
- remove useless fonts and blocking style
- minify css ressource
- add images sizes to avoid HTML/CSSOM reflow - repaint events
- reorganize JS files

I think we can go further with a little bit of time and tests, but the Google Lighthouse standards give use good results for now (keeping in mind it's local)

<img width="872" alt="Capture d’écran 2023-06-23 à 14 20 54" src="https://github.com/gnolang/www.gno.land/assets/7622257/820b0a26-af5e-4222-80cd-0d0e1a426236">